src/Member.java

```diff
@@ -0,0 +1,189 @@
+    private String password;
+    private String role;
+    static ArrayList<Member> userList = new ArrayList<Member>();
+
+    public enum Role {
+        ADMIN, USER
+    }
+
+    public Member() {
+//        this.userName = "tab";
+//        this.password = "123";
+    }
+
+    public Member(String userName, String password, Role role) {
+        this.userName = userName;
+        this.password = password;
+        this.role = role.toString();
+    }
+
+    public Member(String userName, String password) {
+        this.userName = userName;
+        this.password = password;
+        this.role = role;
+    }
+
+    public String getUserName() {
+        return userName;
+    }
+
+    public void setUserName(String userName) {
+        this.userName = userName;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public String getRole() {
+        return role;
+    }
+
+    public void setRole(String role) {
+        this.role = role;
+    }
+
+    public void addSomeDefaultAdmin(String name, String pass) {
+        Member members = new Member(name, pass, Role.ADMIN);
+        userList.add(members);
+    }
+
+    public void addSomeDefaultMembers(String name, String pass) {
+        Member members = new Member(name, pass, Role.USER);
+        userList.add(members);
+    }
+
+    public void addNewUser(String newUser_Name, String newUser_Password) {
+        Member members = new Member(newUser_Name, newUser_Password, Role.USER);
+
+        //userList = new ArrayList<Member>();
+        if (userList.contains(newUser_Name)) {
+            System.out.println("User already Exists !! Please try another user name.");
+        } else {
+            userList.add(members);
+            System.out.println("New Member ["+newUser_Name.toUpperCase()+"] has been added to the Library");
+        }
+
+    }
+
+    public void addNewAdmin(String newAdmin_Name, String newAdmin_Password) {
+        Member members = new Member(newAdmin_Name, newAdmin_Password, Role.ADMIN);
+
+        //userList = new ArrayList<Member>();
+        if (userList.contains(newAdmin_Name)) {
+            System.out.println("User already Exists !! Please try another user name.");
+        } else {
+            userList.add(members);
+            
+        }
+    }
+
+    public void login(String logUserName, String logPassword) {
+        for (int i = 0; i <= userList.size(); i++) {
+            if (logUserName.equals(userList.get(i).getUserName()) && logPassword.equals(userList.get(i).getPassword())) {
+                String logRole = userList.get(i).getRole();
+                System.out.println(logRole);
+                if (logRole.equals("ADMIN")) {
+                    //todo some code
+                    System.out.println("Logged in as ADMIN Successfully !!");
+                    String inputFromAdmin;
+                    do {

```

Comment:
do statements are really old hat although still supported in Java they are as good as dead.

The better solution is to have a while statement

i.e
while(condition){
  //do code
}